src/device/device-management.ts

```diff
@@ -109,7 +109,7 @@ export async function createMicrophoneTrack(
  * @returns A Promise that resolves to a LocalDisplayTrack.
  */
 export async function createDisplayTrack(
-  constraints?: VideoConstraints,
+  constraints?: CameraConstraints,
```

Comment:
Sorry, maybe I missed this when I made my earlier comment, but if the constraints here are generic for all video (camera and display) we should probably stick with `VideoConstraints`.  Same for audio: if they apply to both microphone and 'computer', then it should just be `AudioConstraints`.

If there _are_ different constraints for camera/display and microphone/computer, then we should have 4 different constraint types.

Reply:
we are going to add a new constrains for display later so it is better to have microphone and camera previoly we were using one which does not make sense now as share has other parameters compared to camera

Reply:
done